neurons/validator.py

```diff
@@ -142,14 +141,23 @@ async def forward_prompt(self):
 
         async def wait_till_next_simulation():
             # wait until the next simulation
-            wait_time = timeout_until(next_iteration) / self.len_simulations
+            wait_time = timeout_until(next_iteration) / len(
+                self.simulation_input_list
+            )
             bt.logging.info(
                 f"Waiting for {wait_time} seconds until the next simulation",
                 "forward_prompt",
             )
             await asyncio.sleep(wait_time)
 
-        for simulation_index in range(self.len_simulations):
+        eth_launch_time = datetime(2025, 5, 19, 2, 0, 0, 0, timezone.utc)
```

Comment:
2025-05-19 14 h UTC (2 PM)

```suggestion
        eth_launch_time = datetime(2025, 5, 19, 14, 0, 0, 0, timezone.utc)
```